bootstrap.sh

```diff
@@ -0,0 +1,7 @@
+sudo apt-get update
+sudo apt-get install -y python
+sudo apt-get install -y python-pip
+sudo apt-get install -y vim
+sudo pip install flask
```

Comment:
Could we pull this out into a `requirements.txt` file and do `pip install -r requirements.txt`?